articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
 
-## Is cross-namespace networking supported?
+No. All resources, including the cluster master nodes, run in your customer subscription. These types of resources are put in a read-only resource group.
 
-Customer and individual project admins can customize cross-namespace networking (including denying it) on a per project basis using `NetworkPolicy` objects.
+### Does the cluster reside in a customer subscription? 
 
-## Can an admin manage users and quotas?
+The Azure Managed Application lives in a locked Resource Group with the customer subscription. Customers can view objects in that RG but not modify them.
 
-Yes. An Azure Red Hat OpenShift administrator can manage users and quotas in addition to accessing all user created projects.
+### Is there any element in ARO shared with other customers? Or is everything independent?
 
-## Can I restrict a cluster to only certain Azure AD users?
+Each Azure Red Hat OpenShift cluster is dedicated to a given customer and lives within the customer's subscription. 
 
-Yes. You can restrict which Azure AD users can sign in to a cluster by configuring the Azure AD Application. For details, see [How to: Restrict your app to a set of users](https://docs.microsoft.com/azure/active-directory/develop/howto-restrict-your-app-to-a-set-of-users)
+### Are infrastructure nodes available?
 
-## Can I restrict users from creating projects?
+On Azure Red Hat OpenShift 4.x clusters infrastructure nodes are not currently available.
 
-Yes. Log in to your cluster as an Azure Red Hat OpenShift administrator and execute this command:
+On Azure Red Hat OpenShift 3.11 clusters infrastructure nodes are included by default.
 
-```
-oc adm policy \
-    remove-cluster-role-from-group self-provisioner \
-    system:authenticated:oauth
-```
+## Upgrades
 
-For more information, see the OpenShift documentation on [disabling self-provisioning](https://docs.openshift.com/container-platform/3.11/admin_guide/managing_projects.html#disabling-self-provisioning).
+###  What is the general upgrade process?
 
-## Can a cluster have compute nodes across multiple Azure regions?
+Patches are applied automatically to your cluster. You do not need to take any action to receive patch upgrades on your cluster.
 
-No. All nodes in an Azure Red Hat OpenShift cluster must originate from the same Azure region.
+Running an upgrade is a safe process to run and should not disrupt cluster services. The SRE can trigger the upgrade process when new versions are available or CVEs are outstanding. Available updates are tested in a staging environment and then applied to production clusters. Following best practices helps ensure minimal to no downtime.
 
-## Are master and infrastructure nodes abstracted away as they are with Azure Kubernetes Service (AKS)?
+Planned maintenance is not prescheduled with the customer. Notifications may be sent via email if communication to the customer is required.
```

Comment:
It's not about communication requirements, but about action required by the customer, right?

Reply:
I think right now the plan about how and when to notify customers is still in flux. I kept mostly the old content here, but I'll add a card to revisit all of the update questions once the team solidifies the process.

Reply:
Per our conversation I deleted the last part of the second sentence.